src/app/@ansyn/imagery/imagery/manager/imagery.component.manager.ts

```diff
@@ -82,12 +82,8 @@ export class ImageryComponentManager {
 	}
 
 	private createMapSourceForMapType(mapType: string): Promise<any> {
-		const relevantMapConfig: IMapConfig = this.config.geoMapsInitialMapSource.find((mapConfig) => mapConfig.mapType === mapType);
-		if (!relevantMapConfig) {
-			throw new Error(`getMapSourceForMapType failed, no config found for ${mapType}`);
-		}
-		const sourceProvider = this.getMapSourceProvider({mapType: relevantMapConfig.mapType, sourceType: relevantMapConfig.mapSource});
-		return sourceProvider.createAsync(relevantMapConfig.mapSourceMetadata);
+		const sourceProvider = this.getMapSourceProvider({mapType, sourceType: this.mapSettings.sourceType });
```

Comment:
add relevent exception "throw new Error(`getMapSourceForMapType failed, no config found for ${mapType}`);"